### DISCUSSION TITLE
I tried to give feedback in Lesson 1

### DISCUSSION THREAD

**Discussion by gonzalo-munillag** [OpenMined/courses#121] [Feedback] [answered]
I tried to give feedback in Lesson 1 with the button on the left down corner (megaphone) but it did not work. (feedback was a thumbs up though haha)

**Comment by yemikifouly** [comment, marked as answer]
Yes. We are having an issue with that button. The issue is known. You can track it [here](https://github.com/OpenMined/openmined/issues/76). 🙂 I will let you know once it is solved so you can give us that thumbs up. 😀
